package.json

```
@@ -17,79 +17,80 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~17.0.0",
-    "@angular/cdk": "^17.3.1",
-    "@angular/common": "~17.0.0",
-    "@angular/compiler": "~17.0.0",
-    "@angular/core": "~17.0.0",
-    "@angular/forms": "~17.0.0",
-    "@angular/material": "^17.3.1",
-    "@angular/material-moment-adapter": "^17.3.1",
-    "@angular/platform-browser": "~17.0.0",
-    "@angular/platform-browser-dynamic": "~17.0.0",
-    "@angular/router": "~17.0.0",
+    "@angular/animations": "18.2.13",
+    "@angular/cdk": "18.2.13",
+    "@angular/common": "18.2.13",
+    "@angular/compiler": "18.2.13",
+    "@angular/core": "18.2.13",
+    "@angular/forms": "18.2.13",
+    "@angular/material": "18.2.13",
+    "@angular/material-moment-adapter": "18.2.13",
+    "@angular/platform-browser": "18.2.13",
+    "@angular/platform-browser-dynamic": "18.2.13",
+    "@angular/router": "18.2.13",
     "@biesbjerg/ngx-translate-extract-marker": "^1.0.0",
-    "@ng-icons/core": "^25.6.1",
-    "@ng-icons/material-icons": "^29.5.1",
-    "@ngrx/component-store": "~17.1.1",
-    "@ngrx/effects": "~17.1.1",
-    "@ngrx/entity": "~17.1.1",
-    "@ngrx/router-store": "~17.1.1",
-    "@ngrx/store": "~17.1.1",
+    "@ng-icons/core": "29.10.0",
+    "@ng-icons/material-icons": "29.10.0",
+    "@ngrx/component": "18.1.1",
+    "@ngrx/effects": "18.1.1",
+    "@ngrx/router-store": "18.1.1",
+    "@ngrx/schematics": "18.1.1",
+    "@ngrx/store": "18.1.1",
+    "@ngrx/store-devtools": "18.1.1",
     "@ngx-translate/core": "^15.0.0",
-    "@nx/angular": "17.3.2",
+    "@nx/angular": "^20.4.2",
     "@vendure/ngx-translate-extract": "^9.0.3",
-    "geonetwork-ui": "2.4.0-alpha.2.8ebef57d4",
-    "rxjs": "~7.8.0",
+    "geonetwork-ui": "^2.4.1",
```

Even if it's less important on non-dev versions, I'd stay on an exact version here. You can achieve that with `--save-exact`
```suggestion
    "geonetwork-ui": "2.4.1",
```